field_records/ubna_2023.csv

```diff
@@ -47,9 +47,15 @@ Date and Time deployed (local) , Date and Time recovered (local) , AudioMoth # ,
 2023-07-11T15:09:00 , 2023-07-17T15:01:00 , F , 012 , Central Pond , 47° 39' 15.762'' N , 122° 18' 1.992'' W , 48000 , Medium , None , None , 4.062 , 3.750 , CT , MB , CT , recover-20230717 , Panasonic Batteries Daytime 0:00-24:00 UTC; Reconfigured for Peace of Mind - Central Pond is noticably lower
-2023-07-17T15:47:00 , (DATE-TIME) , H , 005 , Telephone Field , 47° 39' 31.122'' N , 122° 17' 18.132'' W , 48000 , Medium , None , None , 4.083 , (VOLTAGE) , CT , MB , MB , (RECOVER-DATE) , Ikea Batteries Daytime 0:00-24:00 UTC;
-2023-07-17T15:47:00 , (DATE-TIME) , B , 003 , Telephone Field , 47° 39' 31.122'' N , 122° 17' 18.132'' W , 192000 , Medium , None , None , 4.075 , (VOLTAGE) , CT , MB , MB , (RECOVER-DATE) , Panasonic Batteries Nighttime 3:06-13:23 UTC;
-2023-07-17T15:29:00 , (DATE-TIME) , G , 002 , Foliage , 47° 39' 19.440'' N , 122° 17' 26.340'' W , 192000 , Medium , None , None , 4.024 , (VOLTAGE) , MB , CT , MB , (RECOVER-DATE) , Panasonic Batteries Nighttime 3:06-13:23 UTC;
-2023-07-17T15:29:00 , (DATE-TIME) , D , 010 , Foliage , 47° 39' 19.440'' N , 122° 17' 26.340'' W , 48000 , Medium , None , None , 4.087 , (VOLTAGE) , MB , CT , MB , (RECOVER-DATE) , Ikea Batteries Daytime 0:00-24:00 UTC;
-2023-07-17T15:12:00 , (DATE-TIME) , E , 001 , Central Pond , 47° 39' 15.762'' N , 122° 18' 1.992'' W , 192000 , Medium , None , None , 4.084 , (VOLTAGE) , CT , CT , MB , (RECOVER-DATE) , Ikea Batteries Nighttime 3:06-13:23 UTC; Central Pond is super low (ankle height at deployment site);
-2023-07-17T15:12:00 , (DATE-TIME) , F , 011 , Central Pond , 47° 39' 15.762'' N , 122° 18' 1.992'' W , 48000 , Medium , None , None , 4.109 , (VOLTAGE) , CT , CT , MB , (RECOVER-DATE) , Ikea Batteries Daytime 0:00-24:00 UTC;
\ No newline at end of file
+2023-07-17T15:47:00 , 2023-07-25T15:47:00 , H , 005 , Telephone Field , 47° 39' 31.122'' N , 122° 17' 18.132'' W , 48000 , Medium , None , None , 4.083 , 3.695 , CT , MB , MB , recover-20230725 , Ikea Batteries Daytime 0:00-24:00 UTC;
+2023-07-17T15:47:00 , 2023-07-25T15:47:00 , B , 003 , Telephone Field , 47° 39' 31.122'' N , 122° 17' 18.132'' W , 192000 , Medium , None , None , 4.075 , 3.629 , CT , MB , MB , recover-20230725 , Panasonic Batteries Nighttime 3:06-13:23 UTC;
+2023-07-17T15:29:00 , 2023-07-25T16:06:00 , G , 002 , Foliage , 47° 39' 19.440'' N , 122° 17' 26.340'' W , 192000 , Medium , None , None , 4.024 , (VOLTAGE) , MB , CT , MB , recover-20230725 , Panasonic Batteries Nighttime 3:06-13:23 UTC; forgot to check vb before battery removal;
+2023-07-17T15:29:00 , 2023-07-25T16:06:00 , D , 010 , Foliage , 47° 39' 19.440'' N , 122° 17' 26.340'' W , 48000 , Medium , None , None , 4.087 , (VOLTAGE) , MB , CT , MB , recover-20230725 , Ikea Batteries Daytime 0:00-24:00 UTC; forgot to check vb before battery removal;
```

Comment:
@ctuguinay : Just checking if (VOLTAGE) is in your pytest cases or it needs to be added in.

Otherwise, the note looks good!